src/poliastro/neos/_init_.py

```diff
@@ -0,0 +1,6 @@
+'''Code related to NEOs
```

Comment:
Does this pass PEP8? I think it's double quotes for docstrings, correct me if I'm wrong.